src/app/views/shared/services/library-menu.service.ts

```diff
@@ -169,6 +172,9 @@ export class LibraryMenuService {
     }, {
       name: 'Global header',
       link: LibraryItemSegmentPaths.globalheader
+    }, {
+      name: 'Date & time',
```

Comment:
...and don't show this either of course ;)

Reply:
...already done, my bad.

Reply:
Same second :-)